File: 2 Semestre/Tópicos Especiais de Sistemas/Projeto/Academia/Professor.cs
public class Professor{
    private string _nome;
    public string Nome {
        get => _nome;
    }
    private DateTime _dataNascimento;
    public DateTime DataNascimento{
        get => _dataNascimento;
    }
    private string _turno;
    public string Turno {
        get => _turno;
        set => _turno = value;
    }

    public Professor(string _nome, DateTime _dataNascimento, string _turno){
        this._nome = _nome;
        this._dataNascimento = _dataNascimento;
        this._turno = _turno;
    }
}

File: 2 Semestre/Desenvolvimento de Software/Projeto/Projeto-Java/src/entities/Professor.java
package entities;
import java.util.List;

import enums.Turno;

public class Professor extends Funcionario {
    private List<Servico> servicosAtendidos;
    private String codigoProfessor;

    public Professor(String nome, String cpf, Turno turno, double salario,
            List<Servico> servicosAtendidos, String codigoProfessor) {
        super(nome, cpf, turno, salario);
        this.servicosAtendidos = servicosAtendidos;
        this.codigoProfessor = codigoProfessor;
    }

    public List<Servico> getServicosAtendidos() {
        return servicosAtendidos;
    }

    public void setServicosAtendidos(List<Servico> servicosAtendidos) {
        this.servicosAtendidos = servicosAtendidos;
    }

    public String getCodigoProfessor() {
        return codigoProfessor;
    }

    public void setCodigoProfessor(String codigoProfessor) {
        this.codigoProfessor = codigoProfessor;
    }
    
    
}
